tokio/src/net/windows/named_pipe.rs

```diff
@@ -0,0 +1,1305 @@
+            self.open_mode,
+            first,
+            winbase::FILE_FLAG_FIRST_PIPE_INSTANCE
+        );
+        self
+    }
+
+    /// Indicates whether this server can accept remote clients or not. This is
+    /// enabled by default.
+    ///
+    /// This corresponds to setting [PIPE_REJECT_REMOTE_CLIENTS].
+    ///
+    /// [PIPE_REJECT_REMOTE_CLIENTS]: https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-createnamedpipea#pipe_reject_remote_clients
+    pub fn reject_remote_clients(&mut self, reject: bool) -> &mut Self {
+        bool_flag!(self.pipe_mode, reject, winbase::PIPE_REJECT_REMOTE_CLIENTS);
+        self
+    }
+
+    /// The maximum number of instances that can be created for this pipe. The
+    /// first instance of the pipe can specify this value; the same number must
+    /// be specified for other instances of the pipe. Acceptable values are in
+    /// the range 1 through 254. The default value is unlimited.
+    ///
+    /// This corresponds to specifying [nMaxInstances].
+    ///
+    /// [nMaxInstances]: https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-createnamedpipea
+    /// [PIPE_UNLIMITED_INSTANCES]: crate::winapi::um::winbase::PIPE_UNLIMITED_INSTANCES
+    ///
+    /// # Panics
+    ///
+    /// This function will panic if more than 254 instances are specified. If
+    /// you do not wish to set an instance limit, leave it unspecified.
+    ///
+    /// ```should_panic
+    /// use tokio::net::windows::NamedPipeOptions;
+    ///
+    /// # #[tokio::main] async fn main() -> std::io::Result<()> {
+    /// let builder = NamedPipeOptions::new().max_instances(255);
+    /// # Ok(()) }
+    /// ```
+    pub fn max_instances(&mut self, instances: usize) -> &mut Self {
+        assert!(instances < 255, "cannot specify more than 254 instances");
+        self.max_instances = instances as DWORD;
+        self
+    }
+
+    /// The number of bytes to reserve for the output buffer.
+    ///
+    /// This corresponds to specifying [nOutBufferSize].
+    ///
+    /// [nOutBufferSize]: https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-createnamedpipea
+    pub fn out_buffer_size(&mut self, buffer: u32) -> &mut Self {
+        self.out_buffer_size = buffer as DWORD;
+        self
+    }
+
+    /// The number of bytes to reserve for the input buffer.
+    ///
+    /// This corresponds to specifying [nInBufferSize].
+    ///
+    /// [nInBufferSize]: https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-createnamedpipea
+    pub fn in_buffer_size(&mut self, buffer: u32) -> &mut Self {
+        self.in_buffer_size = buffer as DWORD;
+        self
+    }
+
+    /// Create the named pipe identified by `addr` for use as a server.
+    ///
+    /// This function will call the [CreateNamedPipe] function and return the
+    /// result.
+    ///
+    /// [CreateNamedPipe]: https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-createnamedpipea
```

Comment:
I think we usually use backticks around this kind of link.
```suggestion
    /// This function will call the [`CreateNamedPipe`] function and return the
    /// result.
    ///
    /// [`CreateNamedPipe`]: https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-createnamedpipea
```

Reply:
All links have been updated.